pkg/flexvolume/block/driver.go

```diff
@@ -115,8 +116,8 @@ const (
 	rateLimitBucketDefault = 5
 )
 
-func newClient(config *Config) (client.Interface, error) {
-	cp, err := configurationProviderFromConfig(config)
+func newClient(config *providercfg.Config) (client.Interface, error) {
+	cp, err := providercfg.NewConfigurationProvider(config)
```

Comment:
Currently `NewConfigurationProvider` calls `Validate()` which validates using the CCM validation hence the CI is failing due to:

```
3238 2018/11/23 14:28:33 OCI FlexVolume Driver version: b87201e1 (b87201e1)
3238 2018/11/23 14:28:33 '/usr/libexec/kubernetes/kubelet-plugins/volume/exec/oracle~oci/oci init' called with []
3238 2018/11/23 14:28:33 Command result: {"status":"Failure","message":"invalid client config: vcn: Required value: VCNID configuration must be provided if configuration for subnet1 is not provided"}
```